### DISCUSSION TITLE
[Switch] Cursor Pointer as default style?

### DISCUSSION THREAD

**Discussion by ariflogs** [mantinedev/mantine#4468] [Feature requests]
<img width="373" alt="Screenshot 2023-06-27 at 12 38 38 PM" src="https://github.com/mantinedev/mantine/assets/43506814/dc0b8934-5754-4599-bbb5-c3c1b47ba2c2">

It feels a bit odd that the switch component doesn't have the `{cursor: 'pointer'}` style option and I didn't find any props to achieve that! 
I tried to do it with CSS but that doesn't seem to work well. 😅

**Comment by rtivital** [comment]
https://mantine.dev/theming/theme-object/#cursortype
